components/base/Footer.vue

```diff
@@ -54,10 +54,7 @@
           </a>
         </li>
         <li class="">
-          <a
```

Comment:
@ashleysyg for a follow on PR - these footer links should probably mostly be `nuxt-link`s and not `a`s